AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/participant-details/participant-details.component.html

```diff
@@ -79,11 +85,16 @@
           {{ participant?.UserName }}
         </div>
       </div>
-
-      <div class="govuk-grid-row vh-mtop" *ngIf="participant?.HearingRoleName !== 'Judge'">
+      <div class="govuk-grid-row vh-mtop">
         <div class="govuk-grid-column-one-third contact-label">Telephone</div>
         <div id="participant-{{ participant?.ParticipantId }}-phone" class="govuk-grid-column-two-thirds wrap">
-          {{ participant?.Phone }}
+          <div *ngIf="!isJudge(participant)">{{ participant?.Phone }}</div>
+          <div *ngIf="isJudge(participant) && (getJudgePhone() !== '' || getJudgeEmail() !== undefined || getJudgeEmail() !== null)">
+            {{ getJudgePhone() }}
+          </div>
+          <div *ngIf="isJudge(participant) && (getJudgePhone() === '' || getJudgeEmail() === undefined || getJudgeEmail() === null)">
```

Comment:
should this be judgePhone? and the ones above be email?
also this is quite long, maybe we could move to ts and create a get with a nice description? and we could test